bitcoin/src/p2p/mod.rs

```diff
@@ -430,8 +431,12 @@ mod tests {
         // Test formatting.
         assert_eq!("ServiceFlags(NONE)", ServiceFlags::NONE.to_string());
         assert_eq!("ServiceFlags(WITNESS)", ServiceFlags::WITNESS.to_string());
-        let flag = ServiceFlags::WITNESS | ServiceFlags::BLOOM | ServiceFlags::NETWORK;
-        assert_eq!("ServiceFlags(NETWORK|BLOOM|WITNESS)", flag.to_string());
+        assert_eq!("ServiceFlags(P2P_V2)", ServiceFlags::P2P_V2.to_string());
+        let flag = ServiceFlags::WITNESS
```

Comment:
It looks like you're testing to_string() for `P2P_V2`.  Just curious why the other service flags aren't added as well, such as `ServiceFlags::COMPACT_FILTERS`.

Reply:
tbh I'm not sure why to_string needs to be tested on each of these to begin with.

Reply:
Oh, see this is asserting https://github.com/rust-bitcoin/rust-bitcoin/pull/4233/files#diff-6da7f79877a94b9d2c85eabbf3fe2cc41e504fe0e495f01f110ffdf0369b42acL153 through L159,  Anyway, might be worth adding the remaining service flags while here.

Reply:
I prefer to leave this PR only for `P2P_V2`. It's just a simple test addition, main motivation was that P2P_V2 was not included into `all`.

Reply:
sure no worries